src/main/resources/templates/fragments/header.html

```diff
@@ -18,27 +18,26 @@
-        <![endif]-->
+        <!--&lt;!&ndash;[if lt IE 9]>-->
+        <!--<script type="text/javascript" src="static/js/html5shiv.js" th:src="@{~/js/html5shiv.js}"></script>-->
+        <!--<script type="text/javascript" src="static/js/respond.min.js" th:src="@{~/js/respond.min.js}"></script>-->
+        <!--<![endif]&ndash;&gt;-->
 
-        <script type="text/javascript" src="static/js/jquery-1.12.2.min.js" th:src="@{~/js/jquery-1.12.2.min.js}"></script>
-        <script type="text/javascript" src="static/js/bootstrap.min.js" th:src="@{~/js/bootstrap.min.js}"></script>
-        <script type="text/javascript" src="http://maps.google.com/maps/api/js?sensor=true"></script>
-        <script type="text/javascript" src="static/js/jquery.inview.min.js" th:src="@{~/js/jquery.inview.min.js}"></script>
-        <script type="text/javascript" src="static/js/wow.min.js" th:src="@{~/js/wow.min.js}"></script>
-        <script type="text/javascript" src="static/js/mousescroll.js" th:src="@{~/js/mousescroll.js}"></script>
-        <script type="text/javascript" src="static/js/smoothscroll.js" th:src="@{~/js/smoothscroll.js}"></script>
-        <script type="text/javascript" src="static/js/jquery.countTo.js" th:src="@{~/js/jquery.countTo.js}"></script>
-        <script type="text/javascript" src="static/js/lightbox.min.js" th:src="@{~/js/lightbox.min.js}"></script>
-        <script type="text/javascript" src="static/js/main.js" th:src="@{~/js/main.js}"></script>
-        <script type="text/javascript" src="static/js/waypoints.min.js" th:src="@{~/js/waypoints.min.js}"></script>
-        <script type="text/javascript" src="static/js/jquery.flexslider.js" th:src="@{~/js/jquery.flexslider.js}"></script>
+        <!--<script type="text/javascript" src="static/js/jquery-1.12.2.min.js" th:src="@{~/js/jquery-1.12.2.min.js}"></script>-->
+        <!--<script type="text/javascript" src="static/js/bootstrap.min.js" th:src="@{~/js/bootstrap.min.js}"></script>-->
+        <!--<script type="text/javascript" src="http://maps.google.com/maps/api/js?sensor=true"></script>-->
+        <!--<script type="text/javascript" src="static/js/jquery.inview.min.js" th:src="@{~/js/jquery.inview.min.js}"></script>-->
+        <!--<script type="text/javascript" src="static/js/wow.min.js" th:src="@{~/js/wow.min.js}"></script>-->
+        <!--<script type="text/javascript" src="static/js/mousescroll.js" th:src="@{~/js/mousescroll.js}"></script>-->
+        <!--<script type="text/javascript" src="static/js/smoothscroll.js" th:src="@{~/js/smoothscroll.js}"></script>-->
+        <!--<script type="text/javascript" src="static/js/jquery.countTo.js" th:src="@{~/js/jquery.countTo.js}"></script>-->
+        <!--<script type="text/javascript" src="static/js/lightbox.min.js" th:src="@{~/js/lightbox.min.js}"></script>-->
+        <!--<script type="text/javascript" src="static/js/main.js" th:src="@{~/js/main.js}"></script>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)